pages/care.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <link rel="stylesheet" href="/style/care.css">
+    <title>Care</title>
+    <link rel="icon" href="../assets/plant-icon.png" type="image/icon">
+</head>
+<body>
+    <header>
+      <h1>Plant Party Fansite</h1>
+      <nav>
+        <ul>
+          <li>
+            <a class="menu-link" href="../index.html">Home</a>
+          </li>
+          <li>
+            <a class="menu-link" href="#">Care</a>
+          </li>
+          <li>
+            <a class="menu-link" href="facts.html">Facts</a>
+          </li>
+          <li>
+            <a class="menu-link" href="gallery.html">Gallery</a>
+          </li>
+        </ul>
+      </nav>
+  </header>
+  <main>
+    <div class="intro">
+      <div id="summary">
+          <p> This is a quick guide on how to take care of your plants!</p>
+      </div>
+      <div id="table-of-contents">
+        <h2>Plant Care</h2>
+        <ol>
+          <li><a href="#Light">Light 🌞</a></li>
+          <li><a href="#Watering">Watering 💦</a></li>
+          <li><a href="#Soil">Soil and Potting 🪴</a></li>
+          <li><a href="#Temperature">Temperature and Humidity 🌡</a></li>
+        </ol>
```

Comment:
Having links that link off to different parts of the page makes this part of your page feel like a navigation section.

To add more structure to this page, I'd suggest adding a `nav` element to this. The docs [say](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes) "A document may have several nav elements, for  example, one for site navigation and one for intra-page navigation."

```html
<nav>
    <ol>
          <li><a href="#Light">Light 🌞</a></li>
          <li><a href="#Watering">Watering 💦</a></li>
          <li><a href="#Soil">Soil and Potting 🪴</a></li>
          <li><a href="#Temperature">Temperature and Humidity 🌡</a></li>
    </ol>
</nav>